pkg/operator/ceph/object/bucket/provisioner.go

```diff
@@ -46,18 +47,22 @@ type Provisioner struct {
 	accessKeyID     string
 	secretAccessKey string
 
+	// RunAsUser cephx user to use on radosgw-admin to create s3 users
+	RunRgwCmdAsUser string
```

Comment:
nit: what about just naming it `CephUserName`?

Reply:
I think `RunRgwCmdAsUser` is more self-explanatory.